manim/mobject/svg/tex_mobject.py

```diff
@@ -50,11 +50,17 @@ class SingleStringMathTex(SVGMobject):
         "type": "tex",
     }
 
-    def __init__(self, tex_string, **kwargs):
+    def __init__(self, tex_string, template = None, **kwargs):
```

Comment:
Let's add Typings
```suggestion
    def __init__(self, tex_string:str , template:str = None, **kwargs) -> None:
```
Make sure you run black at last. I could be wrong is typing though. Please check.

Reply:
I think you want `tex_template` and not `template` as the kwarg. no?
and then set self.template=kwarg.get('tex_template',config["tex_teamplate]) or sth like that

I am really confused right now about moving template from CONFIG to kwarg because now we have both template and tex_template as kwargs ... both being set to self.template??